Mathlib/Algebra/RingQuot.lean

```diff
@@ -37,6 +37,18 @@ instance (c : RingCon A) : Algebra S c.Quotient where
   commutes' _ := Quotient.ind' fun _ ↦ congr_arg Quotient.mk'' <| Algebra.commutes _ _
   smul_def' _ := Quotient.ind' fun _ ↦ congr_arg Quotient.mk'' <| Algebra.smul_def _ _
 
+variable (S) in
+/-- The algebra morphism from `A` to the quotient by a ring congruence. -/
+def mk'ₐ (c : RingCon A) : A →ₐ[S] c.Quotient := {
+  mk' c with commutes' _ := rfl }
+
+theorem mk'ₐ_apply {c : RingCon A} {a : A} :
+    mk'ₐ c (S := S) a = mk' c a := rfl
```

Comment:
I think `_apply` is automatically generated if you tag the definition with `@[simps]`, and in fact it shows up with the correct right hand side, which is `↑a` rather than `mk' c a`.